src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -64,6 +64,15 @@ def list_subvolumes(self):
                 return []
             raise
 
+    def check_subvolumes(self):
```

Comment:
Could this be renamed to something more meaningful like `is_empty` or any other ?

Reply:
or `has_subvolumes` ?